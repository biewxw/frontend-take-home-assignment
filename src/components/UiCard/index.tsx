import React, { FC } from 'react';
import { Box } from '@chakra-ui/react';

const UiCard: FC = ({ children }) => (
  <Box
    bg="white"
    width="22rem"
    boxShadow="base"
    borderWidth="0.5px"
    borderColor="gray.200"
    borderRadius="1rem"
  >
    {children}
  </Box>
);

export default UiCard;
