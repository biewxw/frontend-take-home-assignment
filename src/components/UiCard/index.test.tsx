import React from 'react';
import UiCard from './index';

import { render } from '@testing-library/react';

describe('<UiCard />', () => {
  it('should render the card with a sample content', () => {
    const { container } = render(<UiCard>That's a sample content</UiCard>);

    expect(container).toMatchSnapshot();
  });
});
