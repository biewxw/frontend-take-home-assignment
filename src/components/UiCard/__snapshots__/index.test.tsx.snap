// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UiCard /> should render the card with a sample content 1`] = `
<div>
  <div
    class="css-lfqrmw"
  >
    That's a sample content
  </div>
</div>
`;
